app/va/va_profile/va_profile_client.py

```diff
@@ -12,10 +14,22 @@
 from time import monotonic
 from typing import Dict, List, TYPE_CHECKING
 
+
 if TYPE_CHECKING:
     from app.models import RecipientIdentifier
     from va_profile_types import ContactInformation, CommunicationPermissions, Profile, Telephone
 
+EMAIL_TYPE = 'email'
+LETTER_TYPE = 'letter'
+MOBILE_TYPE = 'mobile'
+PUSH_TYPE = 'push'
+SMS_TYPE = 'sms'
```

Comment:
These are already defined [here](https://github.com/department-of-veterans-affairs/notification-api/blob/master/app/models.py#L40).  They should be imported.

Reply:
@k-macmillan requested I remove the import and just declare it again in this file.

> With very few exceptions, imports should not happen inside functions. The use case for this dictionary is only in this file. Please remove the import and declare it in this file/class. It is specific to VA Profile and I'm not sure why it's in models (no need to change the other code outside your PR). We will remove it from app.models when we clean up the old get_is_communication_allowed_api method.

Reply:
I guess I did alter the old get_is_communication_allowed function to use the redeclared dict.  I'll revert that part.

Reply:
@k-macmillan What is the reason for not importing?